### PR TITLE
Adds a Unathi hiss emote.

### PR DESCRIPTION
**Pull request by TravellingMerchant**
Adds a hissing emote that sounds like a large lizard, or crocodile. 

https://www.youtube.com/watch?v=O3icLHTmLcE&feature=youtu.be

Original sound from http://www.freesound.org/people/Jamius/sounds/41531/ with the "attack" game like sound effect edited out. It is free to use and edit.

It is used with *hiss and *hisses, with its *hiss-(mob) line too.

Prints "??? lets out a powerful hiss" and "??? lets out a powerful hiss at ???" in chat, respectively.

### CODE REVIEW

**Comment by taukausanake** [comment]
Sounds a little deep. Maybe look for other possible sounds too?

**Comment by KasparoVy** [comment]
Sounds great, pretty tidy compared to the other sounds that've been around too. AFAIK if you want pitch variance or whatever (random between a range) you can handle it like screams used to be done (or have it age dependent, idk) but as is I think it's lit.

**Comment by Vivalas** [comment]
Wildly off topic but I love your honk videos even though I have literally no idea who you are other than from making those.

**Comment by TravellingMerchant** [comment]
Haha, thanks.

**Comment by Fox-McCloud** [comment]
Needs to be updated to acomodate for @KasparoVy's refactor.

File: code/modules/mob/living/carbon/human/emote.dm [hunk 1]
```diff
@@ -210,6 +216,23 @@
...
+					if(param == A.name)
+						M = A
+						break
+			if(!M)
+				param = null
+
+			if(param)
+				message = "<B>[src]</B> lets out a powerful hiss at [param]."
+			else
+				message = "<B>[src]</B> lets out a powerful hiss."
```
Comment (Crazylemon64) [inline code/modules/mob/living/carbon/human/emote.dm]: "lets out a powerful hiss" sounds weird. Can't it just be `[src] hisses`?

File: code/modules/mob/living/carbon/human/emote.dm [hunk 2]
```diff
@@ -210,6 +216,23 @@
...
+			playsound(loc, 'sound/effects/unathihiss.ogg', 50, 0) //Credit to Jamius (freesound.org) for the sound.
```
Comment (Crazylemon64) [inline code/modules/mob/living/carbon/human/emote.dm]: I had concerns about the sound, but I think it'll work?

**Comment by Crazylemon64** [comment]
Please update!

**Comment by Fox-McCloud** [comment]
Last chance. Please update this, or it'll be closed in 2 days.

**Comment by KasparoVy** [comment]
@TravellingMerchant 🐍

**Comment by TravellingMerchant** [comment]
Thanks I'll get to it now.

**Comment by Fox-McCloud** [comment]
Uhhhh, I don't think something went right.

**Comment by TravellingMerchant** [comment]
Oh dear. Uhhh... Im not sure whats wrong.

**Comment by KasparoVy** [comment]
Oh God what the hell, bahaha.
Back up `sound/effects/unathihiss.ogg`, then enter `git fetch upstream` in the git terminal and then try `git reset --hard origin/master`. That will reset the branch to be the exact same as our codebase RN. Then you can drop in the `sound/effects/unathihiss.ogg` and that file from the .zip archive and it should be fine w/ no conflicts

**Comment by KasparoVy** [comment]
@TravellingMerchant Oh God what the hell, bahaha. I think I know what went down

Back up `sound/effects/unathihiss.ogg` somewhere, then enter `git fetch upstream` in the git terminal and then try `git reset --hard origin/master`. That will reset the branch to be the exact same as our codebase RN. Then you can drop in the `sound/effects/unathihiss.ogg` you backed up earlier and that file from the .zip archive can replace `code/modules/mob/living/carbon/human/emote.dm` and it should be fine w/ no conflicts

**Comment by TravellingMerchant** [comment]
Yeah I cant do any of that.

**Comment by KasparoVy** [comment]
Is there any way for you to update this branch from master in the web client?

**Comment by TravellingMerchant** [comment]
I am not sure.

**Comment by KasparoVy** [comment]
Is there any sort of git client you can install on your workstation? Github desktop, Sourcetree.. Anything?

**Comment by Fox-McCloud** [comment]
Stale; closing. Not opposed to the idea, conceptually, but this has sat here for a while. 

May want to contact a number of the coders in teh coderchat if you're having issues with github, or consult a guide.

**Pull request closed without merging**